ibis/backends/base/sql/alchemy/query_builder.py

```diff
@@ -96,8 +96,11 @@ def _format_table(self, op):
             result = ref_op.sqla_table
         elif isinstance(ref_op, ops.UnboundTable):
             # use SQLAlchemy's TableClause for unbound tables
-            result = sa.table(
-                ref_op.name, *translator._schema_to_sqlalchemy_columns(ref_op.schema)
+            result = sa.Table(
```

Comment:
Unfortunately the lighter-weight `sa.table` doesn't accept a `quote` argument so we can't continue to use it here.